src/megaapi_impl.cpp

```diff
@@ -7359,6 +7378,105 @@ bool MegaApiImpl::hasToForceUpload(const Node &node, const MegaTransferPrivate &
     return canForceUpload && (isMedia || isPdf) && !(hasPreview && hasThumbnail);
 }
 
+#ifdef ENABLE_SYNC
+void MegaApiImpl::resumeSyncImpl(const char *localFolder, const long long localfp, const handle nodeHandle, MegaRegExp *regExp, MegaRequestListener *listener)
```

Comment:
This function contains the same body as the original `MegaApiImpl::resumeSync` function.